### DISCUSSION TITLE
How to properly update versions?

### DISCUSSION THREAD

**Discussion by FBosler** [projen/projen#2213] [Q&A] [answered]
First off, amazing package. Secondly, I do have a question around how to update versions. I did try to update versions in the package.json, but encountered errors when trying to merge in this step:

```
👾 release | git diff --ignore-space-at-eol --exit-code
diff --git a/package.json b/package.json
index 11344ed..7d18584 100644
--- a/package.json
+++ b/package.json
@@ -71,7 +71,7 @@
   ],
   "main": "lib/index.js",
   "license": "Apache-2.0",
-  "version": "0.0.2",
+  "version": "0.0.0",
   "jest": {
     "testMatch": [
       "<rootDir>/src/**/__tests__/**/*.ts?(x)",
👾 Task "release" failed when executing "git diff --ignore-space-at-eol --exit-code" (cwd: /__w/apigateway-sqs-lambda/apigateway-sqs-lambda)
Error: Process completed with exit code 1.
```

I am quite frankly not quite sure where I am going wrong here as a lot of the typical pain is abstracted away.

Would also be totally fine with a pointer to documentation around best practices for `projen` projects.

**Comment by mrgrain** [comment, marked as answer]
Hi @FBosler and welcome to projen! 😀 

Have a look at the release docs here: https://projen.io/releases.html

Particular:
- https://projen.io/releases.html#how-can-i-force-a-different-version and
- https://projen.io/releases.html#why-is-the-version-in-packagejson-set-to-000
